files/nutshell/scss/base/_type.scss

```diff
@@ -136,18 +136,29 @@ html {
     $line-height: map-get($options, line-height);
     $margin-top: map-get($options, margin-top);
     $margin-bottom: map-get($options, margin-bottom);
-    $color: map-get($options, color);	  
-    
+    $color: map-get($options, color);
+    $font-weight: map-get($options, font-weight);
+    $font-family: map-get($options, font-family);
     
     font-size: map-get($sizes, xs); // [5]
-    font-family: var(--headings__font-family);
-    font-weight: var(--headings__font-weight);
     line-height: $line-height;
     margin-bottom: $margin-bottom;
-	  
+    
     @if $color {
       color: $color;
+    }
+    	  
+    @if $font-weight {
+      font-weight: $font-weight;
+    } @else {
+      font-weight: var(--headings__font-weight);
     }	  
+	  
+    @if $font-family {
+      font-family: $font-family;
+    } @else {
+      font-family: var(--headings__font-family);
```

Comment:
I think about removing the fallback, as we already have a fallback in line 116ff. What do you think?